### DISCUSSION TITLE
[Projects Beta] Progress Bar

### DISCUSSION THREAD

**Discussion by mdrummond1** [community/community#8887] [Projects and Issues] [answered]
The repository projects board has the option to show progress.
That would be nice here as well.

**Comment by AFPIL** [comment]
is this suggestion similar to https://github.com/github/feedback/discussions/7575 ?

**Comment by lewri** [comment, marked as answer]
For clarification @mdrummond1 are you referring to these?
![image](https://github.com/user-attachments/assets/11b5eeb9-e5ad-4c14-aee1-5e27faa9f1a1)

I also liked these on Projects (Classic) as they gave a nice instant preview at how a project was progressing (both on the projects page, and in issues and PRs)

**Reply by mdrummond1** [reply]
To be honest, this was 4 years ago, and I'm not entirely sure what I was specifically referring to, but it was something similar to that.

I think just more graphical visualizations would be helpful. Maybe a progress bar on the project update sidebar that shows some of the connected repo stats. 

![image](https://github.com/user-attachments/assets/8d334a02-7e87-42f0-9d64-dab22a1a6e86)

Some sort of status indicator here would be helpful as well:
![image](https://github.com/user-attachments/assets/79776a4d-ef59-4a5b-a415-50a8f86eee2c)

And maybe something at the user / organization level that can show change and updates to projects over time.

I know there are also the charts and insights that can be generated. That is something that I have yet to explore
